### PR TITLE
[Moved to different issues] Ethical considerations

### PR DESCRIPTION
**Pull request by nalessandroni**
We created a chapter on ethical considerations (work in progress)

### What should a reviewer concentrate their feedback on?
- [X] This is a product of the book dash and needs more work before it can be reviewed
- [x] Everything looks ok?

### Acknowledging contributors
@Chrisisour @kessonovich @biotechchat @kdixey @nalessandroni
  
- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [X] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: @Chrisisour @kessonovich @biotechchat @kdixey @nalessandroni

### CODE REVIEW

File: book/content/Ethical_Decisions/ethical_considerations.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+- Research is, ultimately, ***a dialogue between what you want, what you can, and what you should***: whenever you get to make a choice, be ***as open as you can***. 
+
+
+# Links to more resources
+- https://www.technologynetworks.com/informatics/articles/faster-progress-for-cancer-research-requires-fairer-access-to-health-and-research-data-305429
+- https://sfdora.org/read/ - includes section on considering all the outputs for research assessment 
+- [Research planning for preclinical scientists](https://github.com/nalessandroni/the-turing-way/blob/master/book/content/Ethical_Decisions/Planning_Research.md)
+
+### Linking to this chapter from existing chapters of the book
+
```
Comment (malvikasharan) [inline book/content/Ethical_Decisions/ethical_considerations.md:375]:
```suggestion
```

**Comment by malvikasharan** [comment]
@nalessandroni @Chrisisour should we go ahead an review this?

**Comment by Chrisisour** [comment]
Hi Malvika,

What does a review entail?

**Comment by nalessandroni** [comment]
Dear Malvika,
I'd be happy to help, please let me know if you need me to fulfill a 
specific task.
I hope you're doing very well.
All my best,

Nicolás.
____________________________________________________________
Nicolás Alessandroni
Depto. de Psicología Evolutiva y de la Educación
Facultad de Psicología
Universidad Autónoma de Madrid (UAM)
Laboratorio 4, C/Iván Pavlov, 6 (28049), Madrid, España
Tlf.: +34.91.497.3285
nicolas.alessandroni@uam.es
https://www.researchgate.net/profile/Nicolas_Alessandroni




------ Mensaje original ------
De: "Malvika Sharan" <notifications@github.com>
Para: "alan-turing-institute/the-turing-way" 
<the-turing-way@noreply.github.com>
CC: "Nicolás Alessandroni" <nicolas.alessandroni@uam.es>; "Mention" 
<mention@noreply.github.com>
Enviado: 08/06/2020 13:45:39
Asunto: Re: [alan-turing-institute/the-turing-way] [WIP] Ethical 
considerations (#913)

**Comment by malvikasharan** [comment]
Hi all, Thank you for your hard work. I am bringing in @Ismael-KG, who have also had a discussion with @Chrisisour. We have decided to split these contributions into different chapters for readability purpose.

@Ismael-KG will add his notes here, and we will start creating new issue to expand those smaller chapters with content from this issue and other relevant resources, examples, case studies, impact stories and data feminism.

@Chrisisour and @nalessandroni, so sorry for such a late response, I wasn't really sure where to begin the ethics part while formatting the book. But hopefully, we can move forward from here on.

**Comment by Ismael-KG** [comment]
Hi all, thanks for bringing me into the conversation @malvikasharan! This is super exciting! After our chat, it seems like the chapters within the ethics book (i.e.: things to create issues for) are:

1. **Landing page** with a definition of ethics, why ethics matters to research (what is research ethics, including mention of the research lifecycle) and a few examples of ethically curious research cases (e.g.: biased facial recognition, crime-predicting algorithms)
2. **Benefits and challenges of ethics within research**: why ethics are important to consider vs. why might it not be possible to consider (I just thought: the way academic institutions are built, consider the publish-or-perish culture)
3. **More abstract philosophical discussion**: (a) very basic discussion around moral philosophy: consequentialism, deontology and virtue ethics, and (b) the inevitable role of epistemic and non-epistemic values
4. **Ethical considerations by research stage**; i.e.: around design, data collection, data storage, analysis, results communication and making open source
5. **Suggested workflows** with different ethics committees' processes
6. **Case studies** (elaborating on ethically curious cases mentioned in intro and sharing more cases)
7. **Impact** of making ethical considerations within science vs. not doing so (@malvikasharan this might actually repeat a bit of point 2... Maybe this issue can be held back? Or do you have a clearer idea as to why this one is necessary)
8. **Law and policy** defining law and distinguishing it from ethics; policies may be mentioned in section 5, so they can be elaborated on here (I also wonder if a list of ethical frameworks could be shared here)

Have I missed anything? And what does everyone think?

**Comment by Chrisisour** [comment]
@Ismael-KG 

Suggestion: Chapter 3 - include ethical values and principles with brief introductions to each value and principle so then in Chapter 4 people are already informed about that

**Comment by Chrisisour** [comment]
_Case studies to be included throughout as examples to illustrate what is said in the chapters_

1. **Landing page** with a definition of ethics, why ethics matters to research (what is research ethics, including mention of the research lifecycle) and a few examples of ethically curious research cases (e.g.: biased facial recognition, crime-predicting algorithms)

2. **Benefits and challenges** of ethics within research: why ethics are important to consider vs. why they might be hard to implement (but always possible to consider) 
2.1. Why to devote resources to ethical processes and how to justify it? Why it is easiest / most important to already design a research project with ethics in mind? Starting research with ethics in mind already makes project planning and management easier

2.2. Standard Q&A about why and how to do ethics, responding to common challenges and criticisms of ethics, misconceptions about ethics and whether it applies to you, e.g.:
- it costs a lot to do ethics - in terms of researcher time to fill in ethics forms. - A: project management-wise you could have 1 person responsible for the form, but ethical thinking should be in the part of decisions, could engage domain ethics experts and bring into discussions, have an Advisory Board, etc. 

2.3. Impact of making ethical considerations within science vs. not doing so (@malvikasharan this might actually repeat a bit of point 2... Maybe this issue can be held back? Or do you have a clearer idea as to why this one is necessary)

3. **Ethical considerations by research stage**; i.e.: around design, data collection, data storage, analysis, results communication and making open source (some of this content can be linked to other parts of the book)
3.1. **ethical design**
- Should we be doing this?
- Who should be doing this? Who is on the team?
- What is the question we are asking?
- What are the ways we could answer that question? Do we always need, e.g. to invade people's privacy? 
- Why are you choosing the method you are choosing? Is it appropriate to answer your question? Are you objective about the results or cherry picking methods that fit your hypothesis?
- How are we planning the data management / data access? - e.g. if you plan to have data being reused, make sure you collect it in the appropriate ways to allow for that (informed consent). Are we working with vulnerable groups of the population, e.g. children, elderly, minority groups and how are you minimising risks? How will you anonymise data before sharing it? (MASSIVE PIECE)

3.2. **ethically conducted research**
- did you actually follow your plan? If not, why not and are you justifying it?
- is your research process auditable, recorded as it took place in practice?
- who is responsible or accountable?

3.3. **ethically distributed research**
- open science in practice - who are you sharing your research with? How are you sharing it?
- data repositories, how are you using those?
- risks of misuse / abuse of your research or your data

4. More abstract philosophical discussion: 
(a) very basic discussion around moral philosophy: consequentialism, deontology and virtue ethics, and 
(b) the inevitable role of epistemic and non-epistemic values
(c) brief (1-2 para) introductions to key ethical values and principles

5. Law and policy 
- define law, policy, and ethics and their relationships / differences
5.1. Law
- what's the difference / relationship between ethics and law
- GDPR and privacy laws (and how privacy and data protection are different)
- HRs law --> check that your national law if there's anything that applies to you / your company
- IP law - ownership or datasets, databases that you are using? T&C of websites when webscraping?
- tort law - if you are negligent / reckless and your work harms someone, you are liable

5.2. Policy on responsible research and innovation (RRI)
- government / international organisations policies (e.g. European Commission guidance for RRI)
- voluntary commitments
- professional bodies, initiatives, statements (e.g. IEEE Ethically-aligned design initiative, ACM explainability and transparency statement)
- a list of resources of Codes of Conduct for ethical research, frameworks, etc. 


_TBD - Suggested workflows with different ethics committees' processes - does it add to what might be in other chapters? Is it something that needs to be done on an institutional basis? of interest to members of ethics committees_

**Comment by Ismael-KG** [comment]
Thank you @Chrisisour @LauraCarter and @brainonsilicon for the revised outline [above](https://github.com/alan-turing-institute/the-turing-way/pull/913#issuecomment-651731815)!

**Comment by malvikasharan** [comment]
Thanks for picking this up @BrainonSilicon, @LauraCarter. 

This PR in its current stage can not be merged. As commented by @Chrisisour and @Ismael-KG, we have refactored this PR into several chapters which have been described in a set of issues. I have updated those issues with a link to this PR  to ensure that the history of contributors is preserved.

Thanks again to the contributors in this PR for helping spin-off several chapters to be added in the Guide for Ethical Research. You are welcome to support the current contributors while they start working on these chapters.

**Pull request closed without merging**